components/hab/src/command/studio/enter.rs

```diff
@@ -43,6 +44,29 @@ fn set_env_var_from_config(env_var: &str, config_val: Option<String>, sensitive:
     }
 }
 
+fn cache_ssl_cert_file(cert_file: &str, cert_cache_dir: &Path) {
+    let cert_path = Path::new(&cert_file);
+
+    if cert_path.exists() && cert_path.is_file() {
+        let cert_filename = match cert_path.file_name() {
+            Some(name) => name,
+            None => {
+                debug!("Unable to determine file name for SSL_CERT_FILE={}",
+                       &cert_file);
+                return;
+            }
+        };
+        let cache_file = cert_cache_dir.join(&cert_filename);
```

Comment:
Since we only execute this code if `cert_path.is_file()` returns true, `cert_path.file_name()` is guaranteed to return `Some(_)`, meaning we can eliminate the whole `match` block and just do:
```rust
        let cache_file = cert_cache_dir.join(cert_path.file_name().unwrap());
```
or, to be a bit more careful:
```rust
        let cert_filename = cert_path.file_name().expect("is file");
        let cache_file = cert_cache_dir.join(cert_filename);
```